AntonBabunin/src/artcode/shop/Bill.java

```diff
@@ -0,0 +1,100 @@
+package artcode.shop;
+
+
+public class Bill {
+    private Product[] products;
+    private double amountPrice;
+    private String seller;
+    private String time;
+    private boolean isClosed = false;
+
+    public Bill(){
+
+    }
+
+//    public Bill(Product[] products, double amountPrice, String seller, String time) {
+//        this.products = products;
+//        this.amountPrice = amountPrice;
+//        this.seller = seller;
+//        this.time = time;
+//    }
+
+    public Product[] getProducts() {
+        return products;
+    }
+
+    public void setProducts() {
+        this.products = new Product[20];
```

Comment:
extract 20 as constant

Reply:
Are you meaning like that ?
+    public void setProducts() {
          int size = 20;
+        this.products = new Product[size];

Reply:
Changed the method logic.